packages/parcel-transformer/src/__tests__/transformer.parceltest.ts

```diff
@@ -15,15 +18,35 @@ afterAll(() => {
 });
 
 const outputFS = new MemoryFS(workerFarm);
+const nodeFS = new NodeFS();
 
-const getParcelInstance = (workingDir: string) => {
+function findTargetSourcePosition(source: string, regex: RegExp): { line: number; column: number } {
+  const lines = source.split('\n');
+
+  for (let i = 0; i < lines.length; i++) {
+    const line = lines[i];
+    const match = line.match(regex);
+
+    if (match && match.index != null) {
```

Comment:
is there a reason loose equality is used here and not strict equality?

Reply:
These are borrowed from a different implementation. I'll refactor to remove the sketchy null comparison